src/modules/map/components/CompareSlider.vue

```diff
@@ -99,16 +100,23 @@ function onPostCompose(event) {
     getMap().render()
 }
 
-function grabSlider() {
+function grabSlider(event) {
     window.addEventListener('mousemove', listenToMouseMove, { passive: true })
     window.addEventListener('touchmove', listenToMouseMove, { passive: true })
     window.addEventListener('mouseup', releaseSlider, { passive: true })
     window.addEventListener('touchend', releaseSlider, { passive: true })
+    compareSliderOffset.value =
+        event.type === 'touchstart'
+            ? event.touches[0].clientX - compareRatio.value * clientWidth.value
+            : event.clientX - compareRatio.value * clientWidth.value
```

Comment:
could you split that into `if...else` statement? it is becoming quite big to be a ternary operator (hard to read what's happening)